### ISSUE TITLE
Questions about the plugin.

### ISSUE THREAD

**Issue by Xag8031** [CyR1en/Minecordbot#106]
I have a Minecraft server I am setting up (for my school), and I was wondering if I could stop people from logging in to my Minecraft server before they are verified. If I ban people from my server on discord would it ban them on my Minecraft server as well or would I need to ban them there? In the features tab under info you say _**"Compatibility with many other chat plugins"**_, is this on discord or in Minecraft? What are the plugins it is compatible with? And what versions does this run on?

**Comment by TheIntelloBox** [comment]
Hello,

You can find the compatibility with many other chat plugins here: https://github.com/CyR1en/Minecordbot/wiki/Compatibility
If I remember correctly, you will need to ban people on Discord and on Minecraft too.
The plugin supports Bukkit 1.8 - 1.12.2.

For more informations: https://github.com/CyR1en/Minecordbot/wiki

Note: The plugin is no longer supported. Last update was 2,5 years ago. 

Best regards,
TheIntelloBox.

**Comment by CyR1en** [comment]
Hello Xag8031,

Minecordbot was a project that I started when I was in my third year of high school (4 years ago). My programming skill at that time was still not up to standards to implement solid features of Minecordbot. At its core, the plugin can only do one thing: to relay message from Discord to Minecraft—and vice versa. As soon as I graduated high school, Minecordbot was pretty much discontinued. All my time just was allocated to college. That’s mainly why features like role-rank synchronization is not a thing in Minecordbot (although there is a build—called M1—on my private server that’s supposed to be the successor of the plugin that had this exact feature; however, this was also abandoned). When it comes to compatibility, Minecordbot was mainly compatible with plugins that modifies the chat system in a different way. Another compatibility is with permission plugins so I could query proper group prefix for each player.

It surprises me that people still reach out to me regarding the plugin when it specifically says it on the plugin page that the project is discontinued (and it has been for many years now). So yeah, there’s really not much that I can do for you at the moment. However, I do recommend checking out plugins like DiscordSRV, etc. It’s a more solid plugin compared to Minecordbot because the dev is actually really skilled (unlike me when I wrote Minecordbot).

Happy new year,
-Cyrien
